imagestreams/mongodb-rhel7-s390x.json

```diff
@@ -0,0 +1,49 @@
+{
+  "kind": "ImageStream",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "mongodb",
+    "annotations": {
+      "openshift.io/display-name": "MongoDB"
+    }
+  },
+  "spec": {
+    "tags": [
+      {
+        "name": "latest",
+        "annotations": {
+          "openshift.io/display-name": "MongoDB (Latest)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Provides a MongoDB database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mongodb-container/tree/master/3.4/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of MongoDB available on OpenShift, including major version updates.",
```

Comment:
```suggestion
          "description": "Provides a MongoDB database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mongodb-container/tree/master/3.6/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of MongoDB available on OpenShift, including major version updates.",
```

Reply:
Hmm would love to commit this but "You do not have permission to push to this repository."

@hhorak Do you perhaps have "Allow edits from maintainers." unchecked?